universal-application-tool-0.0.1/app/views/admin/questions/QuestionsListView.java

```diff
@@ -257,29 +257,32 @@ private Tag renderActionsCell(
       DeletionStatus deletionStatus,
       Http.Request request) {
     ContainerTag td = td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.TEXT_RIGHT);
-    if (active.isPresent() && draft.isEmpty()) {
-      td.with(renderQuestionViewLink(active.get(), "View →")).with(br());
-      td.with(renderQuestionEditLink(active.get(), "New Version →")).with(br());
-    } else if (active.isEmpty() && draft.isPresent()) {
+    if (active.isPresent()) {
+      if (draft.isEmpty()) {
+        // Active without a draft.
+        td.with(renderQuestionViewLink(active.get(), "View →")).with(br());
+        td.with(renderQuestionEditLink(active.get(), "New Version →")).with(br());
+      } else if (draft.isPresent()) {
+        // Active with a draft.
+        td.with(renderQuestionViewLink(active.get(), "View Published →")).with(br());
+        td.with(renderQuestionEditLink(draft.get(), "Edit Draft →")).with(br());
+        td.with(renderQuestionTranslationLink(draft.get(), "Manage Draft Translations →"))
+            .with(br());
+        td.with(renderDiscardDraftLink(draft.get(), "Discard Draft →", request)).with(br());
+      }
+    } else if (draft.isPresent()) {
+      // First revision of a question.
       td.with(renderQuestionEditLink(draft.get(), "Edit Draft →")).with(br());
       td.with(renderQuestionTranslationLink(draft.get(), "Manage Translations →")).with(br());
-    } else if (active.isPresent() && draft.isPresent()) {
-      td.with(renderQuestionViewLink(active.get(), "View Published →")).with(br());
-      td.with(renderQuestionEditLink(draft.get(), "Edit Draft →")).with(br());
-      td.with(renderQuestionTranslationLink(draft.get(), "Manage Draft Translations →")).with(br());
-    } else if (active.isEmpty() && draft.isEmpty()) {
```

Comment:
This state wasn't possible given the checks in the singular caller, it also doesn't make sense to handle this way afaik, `definition` is provided as one of active/draft so this is a little disconnected anyway.